weblogic-azure-aks/src/main/arm/createUiDefinition.json

```diff
@@ -319,6 +331,52 @@
                                 },
                                 "visible": "[not(bool(steps('section_aks').clusterInfo.createAKSCluster))]"
                             },
+                            {
+                                "name": "aksSupportedVersionTextBlock",
+                                "type": "Microsoft.Common.TextBlock",
+                                "visible": "[bool(steps('section_aks').clusterInfo.createAKSCluster)]",
+                                "options": {
+                                    "icon": "Info",
+                                    "text": "AKS supports a range of Kubernetes versions.  This offer is tested with with a specific Kubernetes version known to work with WebLogic server on AKS; click 'Learn more' to the find the version information.",
+                                    "link": {
+                                        "label": "Learn more",
+                                        "uri": "https://aka.ms/wls-aks-well-tested-version"
+                                    }
+                                }
+                            },
+                            {
+                                "name": "useLatestSupportedAksVersion",
+                                "type": "Microsoft.Common.OptionsGroup",
+                                "label": "Use latest supported AKS Kubernetes version",
+                                "defaultValue": "Yes",
+                                "toolTip": "If yes, use the latest supported AKS Kubernetes version; if no, input a version using the known supported versions listed in the document linked above.",
+                                "constraints": {
+                                    "allowedValues": [
+                                        {
+                                            "label": "Yes",
+                                            "value": "true"
+                                        },
+                                        {
+                                            "label": "No",
+                                            "value": "false"
+                                        }
+                                    ],
+                                    "required": true
+                                },
+                                "visible": "[bool(steps('section_aks').clusterInfo.createAKSCluster)]"
+                            },

```

Comment:
Per 2022-05-11 stand up, remove this.

Reply:
See https://github.com/oracle/weblogic-azure/pull/140/commits/2a11a3765fdc133b1237e7bc4e7a8c62296d404f